### DISCUSSION TITLE
design choice for the doxygen generated documentation

### DISCUSSION THREAD

**Discussion by nicolaslg** [LIHPC-Computational-Geometry/gmds#56] [Ideas]
I do not think putting the automated doxygen documentation in the source tree is a good idea; at least not with the current policy (triggered by push on main and pull_requests):
it will likely lead to a bloated repo and lots of bot-authored commits.

I would suggest pushing to a documentation-repo outside of the project, or drastically reduce the frequency of the documentation updates (could even be manual, or when making a release).
@HexMsh : What's your take on it ?

**Comment by franck-ledoux** [comment]
We could indeed push the documentation in another repo or change the policy to update the documentation only for tagged or released versions. 

I just want to be sure that it is built and distributed automatically and not manually.Otherwise the documentation will not be up-to-date or not pushed.
